src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +14,17 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+
+        User foundUser = userService.findByEmail(email);
+
+        if (foundUser == null) {
+            return false;
+        } else if (foundUser.getEmail().equals(email) && foundUser.getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
collapse your if else statement
```suggestion
return foundUser != null && foundUser.getEmail().equals(email) &&
```